### ISSUE TITLE
Touch-swiping problems with Blink-based browsers

### ISSUE THREAD

**Issue by Sigma-90** [malihu/malihu-custom-scrollbar-plugin#355]
I noticed this when I tried to use the plugin to create responsive tables for a web project, where tables that are too large for mobile screens to display are wrapped with a horizontal scrollbar. Using this plugin seemed like the best choice because native scrollbars are invisible on mobile devices and so it isn't immediately obvious to the user that the table actually continues on the right.

The problem I noticed during my testing phase is that in browsers that use the Blink engine (i.e. Chrome for Andrioid and Opera Mobile), vertical swipes are blocked in containers with a horizontal scroll bar, effectively blocking access to all parts of the page below the table unless the user manages to squeeze his finger into that very narrow margin to the left and right of the container. The default Android browser and Firefox Mobile had no such problems and neither had Safari on iOS.

I hope you can come up with a solution to this, as breaking the user experience in Chrome Mobile can be a major deal-breaker for many, I would assume.

**Comment by malihu** [comment]
Which [plugin version](https://github.com/malihu/malihu-custom-scrollbar-plugin/releases) are you using? Can you try with the latest one?

**Comment by Sigma-90** [comment]
I discovered it intially with version 3.0.5 but upgraded to and tested it with 3.1.0 before reporting this issue. I just tried it with 3.1.12 (by the way, it seems the version number was only updated in the minified version, the other one still says 3.1.0) but that didn't fix it, still can't swipe-scroll the page when I'm over the horizontally scrollable element in Chrome.

**Comment by Raolibec** [comment]
yes i get the same problem too. and the problem might be not fixed yet. is there any solutions came out from years @Sigma-90 ?
now it worked well in desktop browsers as well as device switching  in developer tool, firefox mobile browser.
but it become unstable in blink based browsers like chrome on smart phone. 
still have not test safari in IOS/MAC.

**Comment by Sigma-90** [comment]
No real solutions found on my end so far, sorry. A couple of hacks using other plugins (like temporarily blocking access to the scrollable area using my own "scroll-swipe-protection" plugin, for example), but nothing that would cause the plugin to behave as desired.
